torch/csrc/autograd/python_variable.cpp

```diff
@@ -224,12 +224,7 @@ PyObject *THPVariable_get_grad(THPVariable *self)
 {
   auto& var = *self->cdata;
   if (!var.grad) {
-#ifdef WITH_CUDA
-    THCPAutoGPU __guard(var.data->getDevice());
-#endif
-    auto grad = var.data->newTensor();
-    grad->resizeAs(*var.data).zero();
-    var.grad = std::make_shared<Variable>(std::move(grad), 0, 1);
+    return Py_None;
```

Comment:
this needs to be `Py_RETURN_NONE;`

All Python return values need to be incref'd, including None. The `Py_RETURN_NONE` does this for you.